builder/steps/gen-dockerfile/test_config.yaml

```diff
@@ -0,0 +1,72 @@
+schemaVersion: "1.0.0"
+commandTests:
+- name: 'Execution Check'
+  # Create a directory that contains an app.yaml and server.js file
+  # that will be used to generate a Dockerfile and .dockerignore file.
+  setup: [['mkdir', '/tmp/gen_dockerfile_test_dir'],
+          ['echo "env: flex" > /tmp/gen_dockerfile_test_dir/app.yaml'],
+          ['echo "runtime: flex" >> /tmp/gen_dockerfile_test_dir/app.yaml'],
+          ['touch', '/tmp/gen_dockerfile_test_dir/server.js']]
+  command: ['node', '/opt/nodejs-runtime-builder/dist/src/main.js',
+            '--app-dir', '/tmp/gen_dockerfile_test_dir',
+            '--runtime-image', 'some-image']
+  # Remove each file separately to ensure the directory has the
+  # expected files in it.
+  teardown: [['rm', '/tmp/gen_dockerfile_test_dir/app.yaml'],
+            ['rm', '/tmp/gen_dockerfile_test_dir/server.js'],
+            ['rm', '/tmp/gen_dockerfile_test_dir/Dockerfile'],
```

Comment:
No.  This tests just makes sure the `/opt/nodejs-runtime-builder/dist/src/main.js` file can be executed with the given flags.  There are unit tests that verify the `Dockerfile` is generated correctly.